ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Image from '#app/components/Image';
+import { createSrcsets } from '#app/lib/utilities/srcSet';
+import getOriginCode from '#app/lib/utilities/imageSrcHelpers/originCode';
+import getLocator from '#app/lib/utilities/imageSrcHelpers/locator';
+import styles from './styles';
+
+const MaskedImage = ({
+  imageUrl,
+  imageUrlTemplate,
+  imageWidth,
+}: {
+  imageUrl: string;
+  imageUrlTemplate: string;
+  imageWidth: number;
+}) => {
+  const { dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  const url = imageUrlTemplate.split('{width}')[1];
+
+  const originCode = getOriginCode(url);
+  const locator = getLocator(url);
+
+  const { primarySrcset, primaryMimeType, fallbackSrcset, fallbackMimeType } =
+    createSrcsets({
+      originCode,
+      locator,
+      originalImageWidth: imageWidth,
+    });
+
+  return (
+    <div
+      css={[
+        styles.maskedImage,
+        isRtl ? styles.linearGradientRtl : styles.linearGradientLtr,
+      ]}
+    >
+      <Image
+        alt=""
+        src={imageUrl}
+        aspectRatio={[16, 9]}
+        srcSet={primarySrcset || undefined}
+        fallbackSrcSet={fallbackSrcset || undefined}
+        mediaType={primaryMimeType || undefined}
+        fallbackMediaType={fallbackMimeType || undefined}
```

Comment:
I think `|| undefined` can be removed here. The return from `createSrcsets` will be `null` for values, so we can just pass in those variable names as is.